data/assets/scene/digitaluniverse/stars_colormap.asset

```diff
@@ -0,0 +1,22 @@
+local assetHelper = asset.require('util/asset_helper')
+
+local BvColorLUT = asset.syncedResource({
+    Name = "Stars Color Table",
+    Type = "HttpSynchronization",
+    Identifier = "stars_colormap",
+    Version = 2
+})
+
+asset.export("BvColorLUT", BvColorLUT)
+
+asset.meta = {
+    Name = "Stars B-V Colormap",
+    Version = "2.0",
+    Description = [[A lookup table that maps a B-V color index to an RGB color. 
+        The B-V values are in the range (-0.4, 2.0) and each line maps a value 
+        in that range to a color]],
+    Author = "Brian Abbott (AMNH)",
```

Comment:
@micahnyc what would be a good author here?

Reply:
I would just put "OpenSpace Team". That's what I've been putting when there isn't a specific author in mind/known.

It might have been Brian in this case but maybe he got it from somewhere else...